code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -200,4 +202,90 @@
 	new /obj/item/weapon/bikehorn(src)
 	new /obj/item/weapon/tool_upgrade/productivity/ergonomic_grip(src)
 	new /obj/item/weapon/tool_upgrade/refinement/laserguide(src)
-	new /obj/item/weapon/tool_upgrade/augment/ai_tool(src)
\ No newline at end of file
+	new /obj/item/weapon/tool_upgrade/augment/ai_tool(src)
+
+// 	 EXPERIMENTAL WEAPON MODS
+//------------------------------------------------
+
+// Double damage at the cost of more recoil and a tripled energy consumption
+/obj/item/weapon/gun_upgrade/mechanism/battery_shunt
+	name = "Moebius \"Thunder\" battery shunt"
+	desc = "This experimental battery shunt is a cutting edge tool attachment which bypasses battery protection circuits to deliver the maximum amount of power in the shortest amount of time."
+	icon_state = "dampener"
+	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_PLASTEEL = 1, MATERIAL_PLATINUM = 1, MATERIAL_URANIUM = 1)
```

Comment:
Platinum is a Bay snowflake material that's not used mostly anywhere. It would be too hard to print those.

Reply:
I actually like this since it means it needs miners to make it. Platinum is the only material that is rarely found in maintenance but miners get plenty of it.